prisma/seed.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Adds seed data to your db
+ *
+ * @link https://www.prisma.io/docs/guides/database/seed-database
+
+*/
+const { PrismaClient } = require("@prisma/client");
+import { migrateAllBreakpoints } from "./seeds/breakpoints";
+import { migrateBreakpointsMin360 } from "./seeds/breakpoints-min-360";
+import { migratePublishedBreakpoints } from "./seeds/published-breakpoints";
+
+const prisma = new PrismaClient();
+
+async function main() {
```

Comment:
the seeds are not working yet but when I start the app I still have breakpoints, what do they do?

Reply:
huh I don't know if those manual mongodb migrations I did make sense here, they were needed but then I completely migrated the data and I think they can be thrown away

>  I still have breakpoints, what do they do?

I am probably misunderstanding the context, but initial breakpoints are feeded into the db when project is created